steely/plugins/mayo.py

```diff
@@ -0,0 +1,44 @@
+'''gives the previous command m a y o n n a i s e'''
+
+import random
+import re
+import string
+
+__author__ = ('iandioch')
+COMMAND = '.mayo'
+
+FORMATS = [
+    'Ara, {}!',
+    '{} ara.',
+    'ara {}.',
+    '{} #mayo4sam',
+    '{} #mayoforsam',
+]
+
+def mayo(message):
+    message += ' '
+    message = message.lower()
+    message = message.replace('the', 'de') # !important
+    message = message.replace('s', 'sh')
+    message = message.replace('sh ', 's ')
+    message = message.replace('shh', 'sh')
+    sentences = re.split('[.,!]', message)
+    out = ''
+    for s in sentences:
+        if random.random() < 0.4:
+            out += random.choice(FORMATS).format(s) + ' '
+        else:
+            out += s + ' '
```

Comment:
This might be nicer with a list and ' '.join

Reply:
yeah but does mayo deserve "nice"

Reply:
that is a solid point